src/api/FileActivities.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @flow
+ * @file Helper for the box File Activity API
+ * @author Box
+ */
+
+import Base from './Base';
+import { PERMISSION_CAN_COMMENT, PERMISSION_CAN_VIEW_ANNOTATIONS, ERROR_CODE_FETCH_ACTIVITY } from '../constants';
+import type { BoxItemPermission } from '../common/types/core';
+import type { ElementsXhrError } from '../common/types/api';
+import type { FileActivity, FileActivityTypes } from '../common/types/feed';
+
+// We only show the latest reply in the UI
+const REPLY_LIMIT = 1;
```

Comment:
Minor, but this should be configurable via a parameter at some point in the future.

Reply:
I think we can configure this in the future should the value need to be dynamic. Currently we don't use any limit other than 1 so I think it's manageable for now.